File: JavaClass/Assignment/barsChart.py
for a in range(5):
    while True:
        number = int(input("Enter a number between 1 and 30: "))
        if 1 <= number <= 30:
            break
        print("Invalid input. Please try again.")
        
    print(f"Number: {number}")
    print("*" * number)
    print()

File: JavaClass/Assignment/BarsChart.java
import java.util.Scanner;

public class BarsChart {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a = 1;
        int b = 1;
        String z = *;
        while (a <= 5)
             a++;{
            System.out.print("Enter a number between 1 and 30: ");
            int number = input.nextInt();
            
            while (number < 1 || number > 30) {
                System.out.print("Invalid input. Enter a number between 1 and 30: ");
                number = input.nextInt();
            }
            
            System.out.println("Number: " + number);
            while ( b <= number)

                     b++;{
                System.out.println("*" *number  );
            }
            System.out.println("\n");
        }
    }
}
